pkgs/servers/nextcloud/packages/generate.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env nix-shell
+#! nix-shell -I nixpkgs=../../../.. -i bash -p nc4nix
+
+set -e
+set -u
+set -o pipefail
+set -x
```

Comment:
```suggestion
set -euo pipefail
```

turning on -x by default can quite spam the terminal log.